src/pointarray.cpp

```diff
@@ -387,10 +506,53 @@ V3d PointArray::pickVertex(const V3d& cameraPos,
                            double longitudinalScale, double* distance,
                            std::string* info) const
 {
+    // NOTE rayOrigin here is not cameraPos but something else
+    // This routine uses cameraPos as the rayOrigin point
```

Comment:
In the previous system, the `rayOrigin` is a rough guess at the depth of similar points the user is looking at in the scene.  In principle this is context-dependent, but on the current master branch it's basically a point on the clicked ray, at the depth of the centre of camera rotation.

The distinction between `cameraPos` and `rayOrigin` is confusing, I think I could clean things up here with a bit of refactoring.